packages/bygger-backend/src/services/utils/formsApiUtils.ts

```diff
@@ -11,15 +11,18 @@ const createHeaders = (accessToken?: string, revisionId?: number) => {
   };
 };
 
-export const removeInnsendingFromForm = (form: Form): Form => {
-  const formProperties = (({ innsending, ...rest }) => rest)(form.properties);
+export const removeInnsendingTypeFromForm = (form: Form): Form => {
```

Comment:
Vurder å legge til kommentar her for å forklare at denne funksjonen er her for å støtte bakoverkompatibilitet og kan fjernes når alle skjemadefinisjoner er oppdatert.